CHANGELOG.md

```diff
@@ -8,7 +8,9 @@
   * Authorization
   * Authentication
 * Configure Kafka _super users_ (`super.users` field in Kafka configuration) 
-* User operator
+* Added new Entity Operator for deploying:
```

Comment:
We should add not that the Topic Operator is deprecated and users should move to entity operator.